Userland/Services/ContentAccessServer/CMakeLists.txt

```diff
@@ -0,0 +1,22 @@
+serenity_component(
+    ContentAccessServer
+    REQUIRED
```

Comment:
let’s not bloat the minimal build config and add `DEPENDS ContentAccessServer` to each `serenity_component()` that requires this instead :^)

Reply:
sure. idk why anyone would want a "minimal build", but that's irrelevant to this discussion